### PR TITLE
Add challenge that uses the Network tab

### LINKED ISSUE #5: Challenge idea: Use the network request tab
Add a json file with a password. Make a `GET` request in JavaScript on this challenge's page load, where you have to inspect the network request to find the password.

### PR DESCRIPTION
**Pull request by exu3**
Hello! :wave:

This PR adds a challenge that uses the Network tab. 🦑 🐟 
It makes a request to `jellyfish-password.json` which contains the password.

Closes #5 

Screen Shot 2022-02-12 at 21 36 32

Screen Shot 2022-02-12 at 21 36 55

### CODE REVIEW

**Comment by exu3** [comment]
Hello @jessicard! Could you please review this PR whenever you get a chance? Thanks!

File: challenges/hard/jellyfish.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+    <title>Hacker Challenge</title>
+    <link rel="stylesheet" href="/style.css">
+  </head>
+
+  <body>
+    <div class="help-section">
+      <div class="help-blurb hidden">
+        <p>Right click in the browser window and click the last option, <strong>"Inspect"</strong>.</p>
+        <p>Open the <strong>"Network"</strong> tab and find the <code>jellyfish-password.json</code>. If you don't see <code>jellyfish-password.json</code>, try refreshing your page.</p>
```
Comment (jessicard) [inline challenges/hard/jellyfish.html:15]: Small clarification here - could this read:

```
If you don't see <code>jellyfish-password.json</code>, try refreshing your page with the inspector open.
```

File: scripts/global.js [hunk 2]
```diff
@@ -22,17 +22,17 @@ window.addEventListener("DOMContentLoaded", (event) => {
 
   //
   // ---------------------------------------------------------
-  // Styles challenge
+  // Network tab challenge
```
Comment (jessicard) [inline scripts/global.js:25]: Ah this looks like it was annoying for you to work around. Doesn't need to be done in this PR, but do you think I should add an issue to move this section out to each individual JS file so people don't have to keep swapping this over when adding a challenge?

**Review by jessicard — APPROVED** [review]
I have no idea how I missed this PR, thank you for pinging me! This is fantastic, what a great addition to the challenge. Thank you! 🙇

File: challenges/hard/jellyfish.html (hunk 1 above)
Comment (jessicard) [reply challenges/hard/jellyfish.html:15]: @exu3 This isn't a big enough deal to block merging the PR so i'm going to merge it now and we can update that later!

File: scripts/global.js (hunk 2 above)
Comment (exu3) [reply scripts/global.js:25]: I think that might help! 

Alternatively, maybe the last challenge could be constant so contributors wouldn't have to edit this at all.

**Pull request merged**